.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
+  "hostRequirements": {
+    "cpus": 2
+  },
+  "waitFor": "onCreateCommand",
+  "updateContentCommand": {
+    "Installing apt packages": "sudo apt-get -yqq update && sudo apt-get install -yqq python3-pip python3-xrootd"
+  },
+  "postCreateCommand": {
+    "Installing python packages": "python3 -m pip install -r requirements.txt"
+  },
+  "customizations": {
+    "codespaces": {
+      "openFiles": []
```

Comment:
if we merge this and add a blank Jupyter file, could we make it auto-open with this?

Reply:
I think so, yes.